topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,373 @@
+>
+> c) A large proportion of DNA polymorphisms in the MTBC are **singletons**, that is, variants present only in a single strain. This adds to the problem of low diversity, since singletons are not informative about tree topology.
+>
+> d) In this workshop, and indeed in many studies of the MTBC, SNPs are called not against H37Rv, but against a reconstructed ancestral genome. This means that the number of SNPs identified does not reflect the evolutionary distance from some random strain like H37Rv, but from the most recent common ancestor of the MTBC. Take a look at Figure 3 in {% cite Goig2018 %} to see how this affects the number of SNPs identified in a genome.
+Below we will see that this has implications for the interpretation of a tree.
+>
+{: .comment}
+
+
+# The alignment
+Aligned DNA or protein sequences are the starting material for phylogenetic inference with molecular data. Here we will make use of the single nucleotide polymorphisms (SNPs) you obtained in the previous tutorials on SNP calling and transmission clusters, reflecting the diversity of 19 MTBC strains and 1 strain of **M. canettii**. The latter is included to have an outgroup, allowing us to root the phylogeny.
+
+Recall that the alignment here was generated using SNPs called from reference-aligned short reads. A frequently used alternative approach to obtain a phylogeny from short read data is to a) assemble the genomes (see the numerous [Galaxy tutorials](https://training.galaxyproject.org/training-material/topics/assembly) on this topic), b) annotate genes, c) extract genes present in all strains (the "core" genes), d) align the core genes. This approach underlies core genome multilocus sequence typing (cgMLST), which is often used to genotype bacterial pathogens (e.g. {% cite Zhou2021 %}).
+
+An alignment of SNPs looks something like this. Each row is a different strain, each column a position in the reference genome. Because this alignment is based on SNPs, it contains only variable positions. Inspecting the alignment is always a good idea, because also in phylogenetics the principle of 'garbage in, garbage out' applies.
+
+![Alignment](./images/MEGA_alignment.png){:width="500"}
+
+## Get the data
+> ### {% icon hands_on %} Hands-on: Obtain your data
+>
+> 1. Make sure you have an empty analysis history. Give it a name.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the following files from [Zenodo](https://zenodo.org/record/6010176) or from the shared data library
+>
+>    ```
+>    https://zenodo.org/record/6010176/SNP_alignment.fasta
```

Comment:
Should be https://zenodo.org/record/6010176/files/SNP_alignment.fasta

Reply:
Changed, thanks!